OpenRA.Mods.Common/Traits/Player/EnemyWatcher.cs

```diff
@@ -23,7 +23,7 @@ class EnemyWatcherInfo : ITraitInfo
 		public readonly int ScanInterval = 25;
 
 		[Desc("Minimal interval in ticks between notifications.")]
```

Comment:
Since you're here could you make this `Minimal ticks in-between notifications.`?